src/core/third-party-tags/ias.ts

```diff
@@ -5,8 +5,8 @@ import type { GetThirdPartyTag } from '../types';
  * https://integralads.com/uk/
  * @param  {} {shouldRun}
  */
-export const ias: GetThirdPartyTag = ({ shouldRun }) => ({
-	shouldRun,
+export const ias: GetThirdPartyTag = () => ({
```

Comment:
It looks like, because of the signature of `GetThirdPartyTag`, even if we don't use the argument we still pass it in `third-party-tags.ts`.

```typescript
ias({ shouldRun: true })
```

Wonder if it's somehow worth changing the type of `GetThirdPartyTag`, or leaving this as-is and just passing in `true` for the value of `shouldRun`?

Reply:
I realised ias doesn't need to be a function if `shouldrun` is always true

Reply:
Good shout.